src/components/Card.js

```diff
@@ -3,10 +3,24 @@ import React from "react";
 const Card = () => {
   return (
     <div>
+    <CardItem>
       <img src="https://rickandmortyapi.com/api/character/avatar/2.jpeg" alt="Hier muss ein Bild sein" />
       <h2> Morty Smith </h2>
+    </CardItem>
     </div>
   );
```

Comment:
I think you don't need the div elements anymore. And If I saw this correctly then your `CardItem` is a `list element` that is not inside of an `ul element`. Maybe you can wrap your `Card` in an `ul` in your `App.js`?

Reply:
That makes so much sense. Thanks a lot! :)